### PR TITLE
Fix how NITFWriteControl checks if image is blocked

### PR DESCRIPTION
**Pull request by JonathanMeans**

### CODE REVIEW

File: six/modules/c++/six.sidd/tests/test_check_blocking.cpp [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+            six::NITFWriteControl::OPT_NUM_COLS_PER_BLOCK, blockSize);
+    writer.getOptions().setParameter(
+            six::NITFWriteControl::OPT_MAX_PRODUCT_SIZE, "110");
+    writer.initialize(container);
+    writer.save(buffers, pathname, std::vector<std::string>());
+
+}
+void writeTwoImages(const six::Data& data, const std::string& pathname,
+        const std::string& blockSize)
+
```
Comment (asylvest) [inline six/modules/c++/six.sidd/tests/test_check_blocking.cpp]: Extra newline

File: six/modules/c++/six.sidd/wscript [hunk 2]
```diff
@@ -1,6 +1,6 @@
 NAME            = 'six.sidd'
 MAINTAINER      = 'adam.sylvester@mdaus.com'
-MODULE_DEPS     = 'scene tiff nitf xml.lite six mem'
+MODULE_DEPS     = 'cli scene tiff nitf xml.lite six mem'
```
Comment (asylvest) [inline six/modules/c++/six.sidd/wscript]: Add this to TEST_DEPS rather than MODULE_DEPS

**Comment by JonathanMeans** [comment]
This is ready

File: six/modules/c++/six.sidd/wscript [hunk 3]
```diff
@@ -1,6 +1,7 @@
 NAME            = 'six.sidd'
 MAINTAINER      = 'adam.sylvester@mdaus.com'
 MODULE_DEPS     = 'scene tiff nitf xml.lite six mem'
+TEST_DEPS      = 'cli six mem'
```
Comment (asylvest) [inline six/modules/c++/six.sidd/wscript]: The TEST_DEPS inherit the MODULE_DEPS so actually all you need in here is `cli`

**Comment by asylvest** [comment]
Just one more nit and then this is good

**Comment by JonathanMeans** [comment]
Done.

**Pull request merged**